contrib/html_demo/JupyterCode/20_upload_ui.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<ul><li>You're probably aware of this, but just fyi: after running the 13 notebook, most of the files in "local_files" don't exist</li><li>Could add section of text what to do next. E.g. "Now that all files are uploaded, we can run the … . This will show that..." etc</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>